### PR TITLE
chore(CODEOWNERS): update rules

### PR DESCRIPTION
**Pull request by thehowl**
Aims to add more CODEOWNERS rules for specific directories.

WIP still. Figuring out some more rules for the `/examples/` directory. Feel free to take a look though.

### CODE REVIEW

**Comment by thehowl** [comment]
Current state: want to finish up adding more rules for the /examples/ directory, and revisit with a fresh mind the new rules added. Good to receive input on the current proposed rulechanges, though.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -28,35 +29,44 @@
...
+/gno.land/                    @moul @zivkovicmilos
+/gno.land/cmd/genesis/        @zivkovicmilos
+/gno.land/cmd/gnokey/         @jaekwon @moul @gfanton
+/gno.land/cmd/gnoland/        @zivkovicmilos @gnolang/devops
+/gno.land/cmd/gnoweb/         @gfanton @thehowl
+/gno.land/pkg/gnoclient/      @zivkovicmilos @leohhhn
+/gno.land/pkg/gnoland/        @zivkovicmilos @gfanton
+/gno.land/pkg/keyscli/        @jaekwon @moul @gfanton
+/gno.land/pkg/log/            @zivkovicmilos @gfanton
+/gno.land/pkg/sdk/vm/         @gfanton @thehowl
```
Comment (moul) [inline .github/CODEOWNERS:41]:
```suggestion
/gno.land/pkg/sdk/vm/         @moul @gfanton @thehowl
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -28,35 +29,44 @@
...
-/gnovm/cmd/gno                @jaekwon @thehowl @harry-hov @moul
-/gnovm/pkg/gnolang            @jaekwon @piux2 @thehowl @moul
-/gnovm/pkg/doc                @thehowl
-/gnovm/pkg/gnomod             @harry-hov
-/gnovm/pkg/integration        @gfanton
-#/gnovm/pkg/gnoenv
-#/gnovm/pkg/repl
+/gnovm/                       @jaekwon @moul @piux2 @thehowl
+/gnovm/stdlibs/               @thehowl
+/gnovm/tests/                 @deelawn @thehowl
```
Comment (moul) [inline .github/CODEOWNERS:49]:
```suggestion
/gnovm/tests/                 @jaekwon @deelawn @thehowl @mvertes 
```

**Review by moul — APPROVED** [review]

**Review by gfanton — APPROVED** [review]
nice !

**Comment by moul** [comment]
@thehowl, it looks good to merge soon.

**Comment by thehowl** [comment]
I added some additional rules for /examples; it should now be good to go.

**Review by deelawn — APPROVED** [review]

**Pull request merged**